robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -572,36 +620,16 @@ public class ShadowAccessibilityNodeInfo {
 
```

Comment:
The return value of `performAction(int, Bundle)` is actually determined by client code or code under test (`View#performAccessibilityAction(int, Bundle)`).  It has nothing to do with the supported actions or properties.  So in the interest of not making assumptions, let's do this:

Add an interface:

    public interface OnPerformActionListener {
      boolean onPerformAccessibilityAction(int action, Bundle arguments);
    }

The shadow should have a member `OnPerformActionListener actionListener` initialized to `null`

Expose a non-`@Implementation` setter:

    setOnPerformActionListener(OnPerformActionListener listener) {
      actionListener = listener;
    }

And have:

    @Implementation
    public boolean performAction(int action, Bundle arguments) {
      if (performedActionAndArgsList == null) {
        performedActionAndArgsList = new LinkedList<>();
      }
      performedActionAndArgsList.add(new Pair<Integer, Bundle>(new Integer(action), arguments));
      return (actionListener != null) ? actionListener.onPerformAccessibilityAction(action, arguments)
          : true;
    }